xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/EditTranslations.xml

```diff
@@ -134,7 +134,7 @@
   #end
   ## Some documents have no default language set. This is the case with documents having scripts, which obviously are
   ## not translatable and more importantly are not written in a specific spoken language. In order to be consistent with
-  ## the language displayed in the top right corner (global.vm) we use 'default' when the default language is not
+  ## the language displayed at the top right corner (global.vm) we use 'default' when the default language is not
```

Comment:
Thanks, but please be aware that this does count to your CGI task, as this fixes a code comment inside a code page, which is not visible in the User Interface.

As you can see, this page is not similar to the other translation pages, as it does not contain translation keys, but actual Velocity code that is doing something. Sure, there is a naming similarity of the page because it ends with `Translations` but, as you can see, that does not mean it is a page with translation keys.